docs/about/license.md

```diff
@@ -1,19 +1,25 @@
 Copyright (c) 2023 innoCampus
 
+<p style="text-align:justify">
```

Comment:
`style="text-align:justify"` scheint in GitHub keinen effekt zu haben. Ist das nötig?

Reply:
Nötig ist das nicht. Ich finde Blocksatz sieht bei längeren Texten besser aus.